scalajslib/worker/1/src/ScalaJSWorkerImpl.scala

```diff
@@ -103,6 +103,7 @@ class ScalaJSWorkerImpl extends ScalaJSWorkerApi {
         .withSemantics(semantics)
         .withModuleKind(scalaJSModuleKind)
         .withESFeatures(scalaJSESFeatures)
+        .withBatchMode(true)
```

Comment:
Hum, you probably don't want this. This will force clearing every time, including upon success, discarding any incremental state.

Reply:
Then I misread the docs! I will remove it!

Reply:
Or the docs where not good enough ;)

Anyway, `ClearableLinker` automatically clears and renews if any exception occurs (including linking errors, but also other crashes). If you add batch mode, it *also* clears every time.

Batch mode can be useful in CIs, but shouldn't be used for iterative development.